.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Docker Build
+
+on:
+  push:
+    branches:
+      - "*"
+  push_request:
+    branches:
+      - "master"
```

Comment:
Why is this only executing on master?

Reply:
I had been following the way the other repos have been set up, which is a bit of a now though. I'll just updated and we can look to align if desired at a later point